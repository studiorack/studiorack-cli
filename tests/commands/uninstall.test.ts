import { expect, test } from 'vitest';
import { cli } from '../shared';
import { RegistryType } from '@open-audio-stack/core';

test('Uninstall package', async () => {
  expect(cli(RegistryType.Plugins, 'install', 'surge-synthesizer/surge')).toMatchSnapshot();
  expect(cli(RegistryType.Plugins, 'uninstall', 'surge-synthesizer/surge')).toMatchSnapshot();
});
