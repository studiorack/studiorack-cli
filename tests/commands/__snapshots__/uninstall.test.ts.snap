// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Uninstall package 1`] = `
┌─────────────────────────┬──────────┬─────────┬────────────┬─────────┬───────────────────────────────┐
│ Id                      │ Name     │ Version │ Date       │ License │ Tags                          │
├─────────────────────────┼──────────┼─────────┼────────────┼─────────┼───────────────────────────────┤
│ surge-synthesizer/surge │ Surge XT │ 1.3.4   │ 2024-08-11 │ gpl-3.0 │ Instrument, Synth, Modulation │
└─────────────────────────┴──────────┴─────────┴────────────┴─────────┴───────────────────────────────┘
`;

exports[`Uninstall package 2`] = `
┌─────────────────────────┬──────────┬─────────┬────────────┬─────────┬───────────────────────────────┐
│ Id                      │ Name     │ Version │ Date       │ License │ Tags                          │
├─────────────────────────┼──────────┼─────────┼────────────┼─────────┼───────────────────────────────┤
│ surge-synthesizer/surge │ Surge XT │ 1.3.4   │ 2024-08-11 │ gpl-3.0 │ Instrument, Synth, Modulation │
└─────────────────────────┴──────────┴─────────┴────────────┴─────────┴───────────────────────────────┘
`;
